kubernetes/template/DeploymentTemplates/script.sh

```diff
@@ -216,10 +227,11 @@ add_azurecli_source()
 download_akse()
 {
     # Todo update release branch details: msazurestackworkloads, azsmaster
-    retrycmd_if_failure 5 10 60 git clone https://github.com/msazurestackworkloads/aks-engine -b azsmaster || exit $ERR_AKSE_DOWNLOAD
+    retrycmd_if_failure 5 10 60 git clone https://github.com/msazurestackworkloads/aks-engine -b azsmaster-akse-releasev0.34.0 || exit $ERR_AKSE_DOWNLOAD
     
     mkdir -p ./bin
-    tar -zxvf aks-engine/examples/azurestack/aks-engine.tgz -C ./bin
+    tar -xf aks-engine/examples/azurestack/aks-engine-patch-v0.34.1-azs-1904-15.gz
```

Comment:
Can we be consistent to use -C to directly put file to ./bin dir.

Reply:
Unziping the file puts everything folder named aks-engine-patch-v0.34.1-azs-1904-15, wanted to have binary alone in bin rather than bin/aks-engine-patch-v0.34.1-azs-1904-15/

Reply:
I see, next time we need to tar.gz just the files, not the directory.